cogs/slashsync/slashsync.py

```diff
@@ -0,0 +1,38 @@
+"""SlashSync
+
+Sync slash commands from the bot to the server.
+"""
+import logging
+
+from redbot.core import checks, commands
+from redbot.core.commands.context import Context
+from redbot.core.utils.chat_formatting import info, success
+from redbot.core.bot import Red
+
+
+class SlashSync(commands.Cog):
+    """Synchronize slash commands on the bot to Discord"""
+
+    def __init__(self, bot: Red):
+        self.bot = bot
+        self.logger = logging.getLogger("red.luicogs.SlashSync")
+        self.bgTask = self.bot.loop.create_task(self.onBotReady())
+
+    async def onBotReady(self):
```

Comment:
Since this coroutine _does_ wait (until the bot is ready), would this sound more accurate?
```suggestion
    async def waitUntilBotReady(self):
```